### PR TITLE
smoke: support `node` image

### PR DESCRIPTION
**Pull request by Desiki-high**
## Relevant Issue (if applicable)
Part of #1460.

## Details
Add test for `node:19.8` in benchmark.

## Types of changes

_What types of changes does your PullRequest introduce? Put an `x` in all the boxes that apply:_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Go over all the following points, and put an `x` in all the boxes that apply._

- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: smoke/tests/tool/container.go [hunk 1]
```diff
@@ -47,9 +54,17 @@ var URL_WAIT = map[string]RunArgs{
...
+		Arg:     "node /src/index.js",
+		Mount: mountPath{
+			source: "tests/texture/node",
+			target: "/src",
+		},
+	},
 }
 
-var supportContainerImages = []string{"wordpress"}
+var supportContainerImages = []string{"wordpress", "node"}
```
Comment (imeoer) [inline smoke/tests/tool/container.go:67]:
```
func SupportContainerImage(image string) bool {
	_, ok := URL_WAIT[image]
        return ok
}
```

File: smoke/tests/tool/container.go [hunk 2]
```diff
@@ -103,14 +126,17 @@ func RunContainerWithBaseline(t *testing.T, image string, containerName string,
...
 	if ok {
-		startTime := time.Now()
 		runUrlWaitContainer(t, image, containerName, args)
-		endTime := time.Now()
-		containerMetic.E2ETime = endTime.Sub(startTime)
 		defer clearContainer(t, image, containerName)
 	}
+
+	endTime := time.Now()
+	containerMetic.E2ETime = endTime.Sub(startTime)
```
Comment (imeoer) [inline smoke/tests/tool/container.go:139]: `containerMetic.E2ETime = time.Since(startTime)`

File: smoke/tests/tool/container.go (hunk 1 above)
Comment (Desiki-high) [reply smoke/tests/tool/container.go:67]: Good point.

**Comment by Desiki-high** [comment]
The `nydus-integration-test` error always happens, I think it is not caused by this pr. cc @imeoer

**Review by imeoer — APPROVED** [review]

**Pull request merged**